Marlin/Marlin_main.cpp

```diff
@@ -2274,11 +2269,13 @@ static void clean_up_after_endstop_or_probe_move() {
     #if ENABLED(DEBUG_LEVELING_FEATURE)
       if (DEBUGGING(LEVELING)) DEBUG_POS("<<< do_probe_move", current_position);
     #endif
+
+    return !probe_triggered;
```

Comment:
I note it returns `true` to indicate "probe was not triggered." If this is meant as an error condition, then the items above should also return `true` to indicate "probe not triggered."